kratos/tests/cpp_tests/utilities/test_divide_triangle_3d_3.cpp

```diff
@@ -0,0 +1,202 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:     BSD License
+//  			 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/checks.h"
+#include "utilities/divide_triangle_3d_3.h"
+
+namespace Kratos
+{
+	namespace Testing
+	{
```

Comment:
We decided to avoid indentation in namespaces. Also that we indent using 4 spaces.